assets/css/need-a-sweet-heart.css

```diff
@@ -0,0 +1,49 @@
+.need-a-sweet-heart-wrapper{
+    height:80vh;
+    width:100%;
+    display: flex;
+    align-items: center;
+    background-image: url('../images/the-dog-white-and-black.jpg');
+    background-size: cover;
+    background-position-y:50%;
+    justify-content: flex-end;
+}
+.need-a-sweet-heart-box{
+    padding:10%;
+}
+.need-a-sweet-heart-box .small-description span{
+    display: block;
+    line-height: 20px;
+}
+.need-a-sweet-heart-box p{
+    text-align: right;
+    margin-bottom: 20px;
+}
+.need-a-sweet-heart-box h1{
+    text-align: right;
+    margin-bottom: 20px;
+}
+.need-a-sweet-heart-box-text-area{
+    display:flex;
+    justify-content:flex-end;
+    align-items: center;
+}
+.need-a-sweet-heart-text-area{
+    width:380px;
+    height:45px;
+    font-size:16px;
+    padding-left:15px;
+    box-sizing: border-box;
+}
+.need-a-sweet-heart-bottom{
+    background:#111;
+    width:120px;
+    height:45px;
+    color:#fff;
+    display:flex;
```

Comment:
Nie jest to złe rozwiązanie, ale w takim przypadku lepiej zrobić inaczej. 
Skoro znasz height (przypisałeś mu wartość 45px) to dodaj tylko line-height: 45px; text-align: center;

Reply:
Poprawione